Products/ResourceRegistries/tools/CSSRegistry.py

```diff
@@ -1,8 +1,7 @@
+from .packer import CSSPacker
```

Comment:
@pbauer would it be possible to use full imports? That makes things easier when refactoring or analyzing packages.

Reply:
-1. I believe many files would get too verbose, potentially uglifying Plone codebase.

It seems to me in this case the problem is in the side of the analysers: why is it difficult for them to figure out the full import path (and give all devs more free time)?

Reply:
@davilima6 what about consistency? mixing relative and full imports is even worse, if you keep, as our style guide says all imports sorted alphabetically, then you have imports from the same package in a complete different location, which developers would have to spend more time hunting them.

Reply:
I agree there must be consistency, for same package imports one should decide to use either relative or absolute imports - a linter would be helpful, as well as a code style guide reckon.

In regard to "findability" I actually like to have a different notation (the starting dot) to quickly spot which imports are "local", and thus likely to have more closely related code. When used consistently, I see that as an advantage.

Reply:
Also maybe the code analysers perform better with the new explicit syntax for relative imports?

Reply:
We usually use absolute imports in Plone. Sometimes relative imports may avoid circular problems, so we need to use them then. In this case I don't see any problems.